app/assets/javascripts/global_listeners.js

```diff
@@ -1,10 +1,21 @@
+import Cookies from 'js-cookie';
 import ExecuteAttachinary from './libs/execute_attachinary';
 import KeycutView from './views/keycut_view';
 
 var $navbar = $(".navbar");
 var $navbarToggle = $('.toggle-navbar.more');
 var $sidebarToggleIcon = $("#sidebar-toggle").children('.mi');
 var $sidebarWrapper = $("#sidebar-wrapper");
+var $cookiesBanner = $(".cookies-banner");
+var $cookiesBannerBtn = $(".cookies-banner__btn");
```

Comment:
To make it a little more precise and indicate that the button is inside the banner, what do you think about doing like this:

```suggestion
var $cookiesBannerBtn = $cookiesBanner.find(".cookies-banner__btn");
```